### PR TITLE
feat: make load-cluster-specs the default

### PR DESCRIPTION
**Pull request by diamonwiggins**
## Description, Motivation and Context

Please include a summary of the change or what problem it solves. Please also include relevant motivation and context.
Screenshot 2024-08-23 at 1 30 40 PM

## Checklist

- [x] New and existing tests pass locally with introduced changes.
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] The commit message(s) are informative and highlight any breaking changes
- [ ] Any documentation required has been added/updated. For changes to https://troubleshoot.sh/ create a PR [here](https://github.com/replicatedhq/troubleshoot.sh/pulls)

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: docs/preflight.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ preflight [url] [flags]
       --as string                      Username to impersonate for the operation. User could be a regular user or a service account in a namespace.
       --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
       --as-uid string                  UID to impersonate for the operation.
-      --cache-dir string               Default cache directory (default "$HOME/.kube/cache")
+      --cache-dir string               Default cache directory (default "/Users/diamonwiggins/.kube/cache")
```
Comment (hedge-sparrow) [inline docs/preflight.md]: Might want to make this more generic

File: docs/support-bundle.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ support-bundle [urls...] [flags]
       --as string                      Username to impersonate for the operation. User could be a regular user or a service account in a namespace.
       --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
       --as-uid string                  UID to impersonate for the operation.
-      --cache-dir string               Default cache directory (default "$HOME/.kube/cache")
+      --cache-dir string               Default cache directory (default "/Users/diamonwiggins/.kube/cache")
```
Comment (hedge-sparrow) [inline docs/support-bundle.md]: did this get expanded by your editor or something?

**Review by hedge-sparrow — CHANGES_REQUESTED** [review]
some minor docs typos?

**Review by hedge-sparrow — APPROVED** [review]

**Review by banjoh — APPROVED** [review]
A welcome change indeed!

LGTM

**Pull request merged**